pkgs/applications/networking/browsers/ephemeral/default.nix

```diff
@@ -0,0 +1,60 @@
+{ stdenv
+, fetchFromGitHub
+, desktop-file-utils
+, gettext
+, glib
+, gtk3
+, hicolor-icon-theme
+, libgee
+, meson
+, ninja
+, pantheon
+, pkgconfig
+, python3
+, webkitgtk
+, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "ephemeral";
+  version = "4.2.0";
+
+  src = fetchFromGitHub {
+    owner = "cassidyjames";
+    repo = "ephemeral";
+    rev = version;
+    sha256 = "1bdqi1k1x2aywi2jm610qil2k5xn2kbs2nni55c5pgilq0sd7xnp";
+  };
+
+  nativeBuildInputs = [
+    desktop-file-utils
+    gettext
+    meson
+    ninja
+    pantheon.vala
+    pkgconfig
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    glib
+    gtk3
+    hicolor-icon-theme
+    libgee
+    pantheon.granite
+    python3
```

Comment:
I'm pretty sure we only need this in `nativeBuildInputs` so `patchShebangs` can give right python executable path in the `post_install.py`.